web-client/src/modules/create/components/PostSummaryStep.tsx

```diff
@@ -12,7 +12,7 @@ import {
   MapDisplay,
```

Comment:
Renamed MyPosts to myRequests as part of splitting post logic into request and offer logic.